doc/manual/src/contributing/hacking.md

```diff
@@ -132,6 +111,26 @@ $ nix-build
 
 You can also build Nix for one of the [supported platforms](#platforms).
 
+## Makefile variables
+
+You may need `profiledir=$out/etc/profile.d` and `sysconfdir=$out/etc` to run `make install`.
+
+Run `make` with [`--environment-overrides`](https://www.gnu.org/software/make/manual/make.html#index-_002de) to allow environment variables to override `Makefile` variables:
+
+- `ENABLE_BUILD=yes` to enable building the C++ code.
```

Comment:
```suggestion
- `ENABLE_BUILD=yes`: build the C++ code.
```

same below

Reply:
I briefly considered this, but then deemed the change too noisy, and it's not that much of an improvement in terms of clarity or readability. I can do it if you think it's a blocker.

Reply:
Not a blocker.